source/index.html.slim

```diff
@@ -23,42 +23,35 @@ title: Home
             h1.hero__slogan
               span.hero__hidden KiwiRuby
               br
-              | The ruby conference you
-              span style="letter-spacing:-.1em" &rsquo;
-              ' ve been looking for.
-              br
-              span.hero__date
-                ' Late 2017 — Wellington NZ
+              p.hero__date
+                ' November 2-3 2017
+              p.hero__date
+                ' Wellington, NZ
     section
       .body-inner-wrapper
         .body-inner-wrapper__section
           h2 Tell me more
           p
-            ' Tautem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut voluptates repudiandae molestiae non. Itaque earum rerum hic tenetur a sapiente delectus.
+            ' KiwiRuby is a new Ruby conference being held in Wellington, New Zealand.
           p
-            ' Itaque earum rerum hic tenetur a sapiente delectus. There are more details to come. Follow us on
-            a> href="https://twitter.com/RubyConfNZ" Twitter
-            ' to stay updated!
-        .body-inner-wrapper__section
-          h2 When? Where?
+            ' With a mixture of local and international speakers, there will be one day of workshops and one single-track day of talks about topics that interest, excite and delight Rubyists.
```

Comment:
I guess mainly I wanted to appeal to web developers and programmers in general who may only have a passing interest in Ruby